tests/test_fixtures/v3/notebook_assets/suite_edit/AUTHORING_INTRO.md

```diff
@@ -0,0 +1,90 @@
+## Create & Edit Expectations
+
+{% if batch_request %}
+Add expectations by calling specific expectation methods on the `validator` object. They all begin with `.expect_` which makes autocompleting easy using tab.
+{% else %}
+You are adding Expectation configurations to the suite.  Since there is no sample batch of data, no validation happens during this process. 
+To illustrate how to do this, consider a hypothetical example.  Suppose that you have a table with the columns "account_id", "user_id", "transaction_id", "transaction_type", and "transaction_amt_usd".
+Then the following code snipped adds an expectation that the columns of the actual table will appear in the order specified above:
+```
+# create an Expectation
```

Comment:
This should mirror the new renderer text - was the test using this fixture passing with this text?

Reply:
@alexsherstinsky Not sure if you saw this message (it was collapsed)

Reply:
Fixed.